src/Parser/EnhancementShaman/Modules/ShamanCore/Pandemic.js

```diff
@@ -0,0 +1,54 @@
+import SPELLS from 'common/SPELLS';
```

Comment:
I would also name this file something else since this module seems to serve solely as a flametongue early refresh checker. Perhaps just name it 'FlametongueRefresh' or something of the sort. It's not considered good practice to have multiple suggestions per module anyway, so you wouldn't want to add Frostbrand here to make it a "Pandemic" module.